This is main metrics, which is my life about. I've defined parameters, which is most important elements in my story.
So I put a mark for each parameter and took average.
I want to practice Python and that's why I created this file.
In the end I count of mark of life satisfaction.

In [1]:
#here is components and marks I've set

art = 1
improving = 9
projects = 7
next_level = 10
nature = 10
heart = 9
curiosity = 3
creating = 7
purposefullness = 10
inspiration = 8

In [3]:
#here is import of lib
import numpy as np

In [4]:
#here is function for average of marks
a = [art, improving, projects, next_level, nature, heart, curiosity, creating, purposefullness, inspiration]
np.average(a)

7.4

My satisfation from life is 7.4 
Not bad, but there is a place for more :)